src-java/kilda-model/src/main/java/org/openkilda/model/HaFlow.java

```diff
@@ -212,10 +212,20 @@ public void setProtectedReversePath(HaFlowPath reversePath) {
         }
     }
 
+    /**
+     * Checks if specified path is protected.
```

Comment:
Some ambiguity here, it could be understood as: a path is protected = meaning this path has a protected path. You probably meant `Checks if the specified path is a protected path` (of this HaFlow)

btw, there is only one protected path for HA Flow?

Reply:
An HA path has an HA path (with 3 endpoints).
Each Ha path consists of 2 common path (with 2 endpoints).

So Ha flow with sw1, sw2, sw3 endpoints has 2 paths:
1. primary: with endpoints sw1, sw2, sw3
2. and protected. with endpoints sw1, sw2, sw3

primary path consist of 2 FlowPaths
1. sw1, sw4, sw2
2. sw1, sw4. sw3

protected path consist of 2 other FlowPaths
1. sw1, sw5, sw2
2. sw1, sw5, sw3

primary and protected must not intersect (excluding endpoints)

Reply:
ah, yes, I thought about a path as two end points thing. When there are 3 end points it explains it. Thanks